inc/class-wpseo-options.php

```diff
@@ -2791,192 +2791,195 @@ protected function clean_option( $option_value, $current_version = null, $all_ol
 /*******************************************************************
  * Option: wpseo_ms
  *******************************************************************/
-if ( is_multisite() ) {
```

Comment:
Added skip for MS only instances in Options constructor in https://github.com/Yoast/wordpress-seo/commit/0decfdeba48d10a4be5cd09ac0de2c911b334bc2